src/main/java/com/faforever/api/data/domain/BanInfo.java

```diff
@@ -43,4 +78,45 @@ public String getReason() {
   public OffsetDateTime getExpiresAt() {
     return expiresAt;
   }
+
+  @Column(name = "type")
+  @Enumerated(EnumType.STRING)
+  public BanType getType() {
+    return type;
+  }
+
+  @Column(name = "create_time")
+  public OffsetDateTime getCreateTime() {
+    return createTime;
+  }
+
+  @Column(name = "update_time")
+  public OffsetDateTime getUpdateTime() {
+    return updateTime;
+  }
+
+  @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+  // Cascading is needed for Create & Delete
+  @JoinColumn(name = "id")
+  public BanRevokeData getBanRevokeData() {
+    return banRevokeData;
+  }
+
+  @Transient
+  public BanDurationType getDuration() {
```

Comment:
`getBanDurationType()` or `getBanType()` (maybe rename `BanDurationType` to `BanType`?)
(see comment at `BanType` as well)